mindsdb_sdk/connectors/rest_api.py

```diff
@@ -37,7 +37,8 @@ def _raise_for_status(response):
 
 
 class RestAPI:
-    def __init__(self, url=None, login=None, password=None, api_key=None, is_managed=False, headers=None):
+    def __init__(self, url=None, login=None, password=None, api_key=None, is_managed=False, 
+                 cookies=None, headers=None):
```

Comment:
what cookie name is going to be used? is it related to authorization?

Reply:
Cookies are used to track session. I'll add you the related PRs.